include/certgen_accessor.h

```diff
@@ -16,15 +17,78 @@
     You should have received a copy of the GNU General Public License along
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-    =========================================================================
+    ========================================================================
 */
 
 #ifndef CERTGEN_ACCESSOR_H_INCLUDED
 #define CERTGEN_ACCESSOR_H_INCLUDED
 
+#include <fty_log.h>
```

Comment:
Log is not needed in the header (not even in cpp as we do not log anything at library level)